utility/windows/DynamicLibrary.cpp

```diff
@@ -41,18 +41,17 @@ const std::string DynamicLibrary::_osLibraryPrefix = "";
 const std::string DynamicLibrary::_osLibrarySuffix = ".dll";
 
 DynamicLibrary::DynamicLibrary(const std::string& path)
+    : _path(osSanitizePathName(path))
 {
-    static_assert(sizeof(void *) == (sizeof(HMODULE), "Incompatible object size");
+    static_assert(sizeof(void *) == sizeof(HMODULE), "Incompatible object size");
 
-    std::string sanitizedPath = osSanitizePathName(path);
-
-    HMODULE module = LoadLibrary(sanitizedPath.c_str()));
+    HMODULE module = LoadLibrary(_path.c_str());
 
     _handle = reinterpret_cast<void *>(module);
 
     if (_handle == nullptr) {
 
-        throw std::runtime_error(sanitizedPath + ": cannot open shared object file.");
+        throw std::runtime_error(_path + ": cannot open shared object file.");
```

Comment:
maybe add the orginal path in the message:
...file. Path generated from: " + path);